packages/ecs-agent/ecs-agent.spec

```diff
@@ -339,44 +192,38 @@ ln -rs %{buildroot}%{_cross_localstatedir}/ecs/managed-agents/execute-command/co
 install -d %{buildroot}%{ecs_exec_dir}/certs
 ln -rs %{buildroot}%{_cross_sysconfdir}/pki/tls/certs/ca-bundle.crt %{buildroot}%{ecs_exec_dir}/certs/tls-ca-bundle.pem
 
-# Prepare license and vendor information so it can be co-installable
-mv %{ecscni_gorepo}-%{ecscni_gitrev}/LICENSE %{ecscni_gorepo}-%{ecscni_gitrev}/LICENSE.%{ecscni_gorepo}
-mv %{vpccni_gorepo}-%{vpccni_gitrev}/LICENSE %{vpccni_gorepo}-%{vpccni_gitrev}/LICENSE.%{vpccni_gorepo}
-# Move vendor folder into a single directory so cross_scan_attribution can run once
-mkdir go-vendor
-mv %{agent_gorepo}-%{agent_gover}/agent/vendor go-vendor/%{agent_gorepo}
-mv %{ecscni_gorepo}-%{ecscni_gitrev}/vendor go-vendor/%{ecscni_gorepo}
-mv %{vpccni_gorepo}-%{vpccni_gitrev}/vendor go-vendor/%{vpccni_gorepo}
-%cross_scan_attribution go-vendor go-vendor
+install -d %{buildroot}{%{_cross_libexecdir},%{_cross_fips_libexecdir}}/amazon-ecs-agent
+# Create symlinks to ECS CNI plugin binaries
+for p in \
+  ecs-bridge \
+  ecs-eni \
+  ecs-ipam \
+; do
+  ln -rs %{buildroot}%{_cross_libexecdir}/cni/ecs/${p} %{buildroot}%{_cross_libexecdir}/amazon-ecs-agent/${p}
+  ln -rs %{buildroot}%{_cross_libexecdir}/cni/ecs/${p} %{buildroot}%{_cross_fips_libexecdir}/amazon-ecs-agent/${p}
+done
+
+# Create symlinks to VPC CNI plugin binaries
+for p in \
+  vpc-branch-eni \
+  aws-appmesh \
+  vpc-eni \
+; do
+  ln -rs %{buildroot}%{_cross_libexecdir}/cni/vpc/${p} %{buildroot}%{_cross_libexecdir}/amazon-ecs-agent/${p}
+  ln -rs %{buildroot}%{_cross_libexecdir}/cni/vpc/${p} %{buildroot}%{_cross_fips_libexecdir}/amazon-ecs-agent/${p}
+done
+
+%cross_scan_attribution go-vendor agent/vendor
 
 install -d %{buildroot}%{_cross_datadir}/logdog.d
 install -p -m 0644 %{S:300} %{buildroot}%{_cross_datadir}/logdog.d
 
 %files
-# License and attribution files are installed into /usr/share/licenses with a
-# directory structure as follows:
-# /usr/share/licenses/ecs-agent/
-# ├── attribution.txt
-# ├── LICENSE
-# ├── LICENSE.amazon-ecs-cni-plugins
-# ├── LICENSE.amazon-vpc-cni-plugins
-# ├── NOTICE
-# ├── THIRD_PARTY.md
-# └── vendor
-#     ├── amazon-ecs-agent
-#     │ └── ...
-#     ├── amazon-ecs-cni-plugins
-#     │ └── ...
-#     └── amazon-vpc-cni-plugins
-#       └── ...
-
 %{_cross_attribution_file}
 %{_cross_attribution_vendor_dir}
-%license %{agent_gorepo}-%{agent_gover}/LICENSE
-%license %{agent_gorepo}-%{agent_gover}/NOTICE
-%license %{agent_gorepo}-%{agent_gover}/ecs-agent/THIRD_PARTY.md
-%license %{ecscni_gorepo}-%{ecscni_gitrev}/LICENSE.%{ecscni_gorepo}
-%license %{vpccni_gorepo}-%{vpccni_gitrev}/LICENSE.%{vpccni_gorepo}
+%license LICENSE
+%license NOTICE
+%license ecs-agent/THIRD_PARTY.md
```

Comment:
You can have one line, instead of three. Probably we did the multi-line `%license` because it was too long. Now it isn't.